imtool-aos/main.cpp

```diff
@@ -25,8 +25,7 @@ int main(int argc, char *argv[]) {
           read_image_intensity_scaling(stoi(args[3]));
       } else if (args[2] == "resize") {
           prog_resize(argc, args);
-          Image_Attributes newImageData = {.width=static_cast<unsigned int>(stoi(args[3])), .height=static_cast<unsigned int>(stoi(args[4]))};
-          aos_resize(newImageData);
```

Comment:
I need this. please remove this change and push again

Reply:
done